pkg/filters/filters.go

```diff
@@ -0,0 +1,98 @@
+package filters
+
+import (
+	"errors"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	LabelFilterName    = "label"
+	AgeFilterName      = "age"
+	KorLabelFilterName = "korlabel"
+)
+
+// KorLabelFilter is a filter that filters out resources that are ["kor/used"] != "true"
+func KorLabelFilter(object runtime.Object, opts *Options) bool {
+	if meta, ok := object.(metav1.Object); ok {
+		if meta.GetLabels()["kor/used"] == "true" {
+			return true
+		}
+	}
+	return false
+}
+
+// LabelFilter is a filter that filters out resources by label
+func LabelFilter(object runtime.Object, opts *Options) bool {
+	if meta, ok := object.(metav1.Object); ok {
+		if has, err := HasExcludedLabel(meta.GetLabels(), opts.ExcludeLabels); err == nil {
+			return has
```

Comment:
if the object matches the exclusion selector shouldn't it be filtered? shouldn't this be `return !has`?